include/rewd_controllers/MoveUntilTouchController.hpp

```diff
@@ -40,10 +40,10 @@ class MoveUntilTouchController final
    */
   void starting(const ros::Time& time) override;
 
-  // inherit documentation
+  // Documentation inherited
   void stopping(const ros::Time& time) override;
 
-  // inherit documentation
+  // Documentation Inherited.
```

Comment:
Nit: `Inherited` -> `inherited`

Reply:
Haha, it seems like I took @gilwoolee 's request for `Documentation Inherited` a little too literal :D